docs/linguistic-data-intro.md

```diff
@@ -133,3 +158,89 @@ by making sure every row has the same number of tabs.
+as familiar with the language, or your conventions for describing it, as
+you are.
+
+This may be in the form of prose documents, spreadsheets, diagrams, and so
+on, as long as the intent is to provide reference material for humans to
+understand what is going on.
+
+  - **Part-of-speech categories** and the relations among them
+
+    Linguists classify language elements into potentially overlapping
+    categories with varying levels of detail, e.g., “nouns” and “verbs” and
+    “type-1 animate intransitive verbs.” They also group these categories
+    into other meta-categories, such as ‘part of speech’ or ‘word class’ or
+    ‘linguistic category.’ For example, they may say that ‘nouns’ and
+    ‘verbs’ are ‘parts of speech’ while ‘transitive verbs’ and
+    ‘intransitive verbs’ are ‘word classes’ within the ‘verb’ ‘part of
+    speech.’
+
+    As handy as it would be, there is no universal terminology here, so
+    please spell out what terminology you’re using!
+
+    What categories do you have, what other categories do they contain or
+    overlap with, and what if anything do you call the different
+    meta-categories you use to group your categories?
+
+The morphodict code only really cares about what it calls a ‘paradigm’: a
+set of words where you’ll get a correct paradigm table output by using a
+template with the same shape, the same labels, and the same FST tags but
+differing entry-dependent FST lemmas for every word in the set.
+
+So your transitive and intransitive verbs are likely to belong to different
+paradigms from morphodict’s perspective because the respective generated
+paradigm table outputs will have different shapes based on whether they
+accommodate an object or not. But whether you have different
+morphodict-paradigms within the intransitive verbs depends on both the
+language and how the FST is implemented.
+
+It’s extremely helpful to know how morphodict’s ‘paradigm’ concept maps to
+the linguistic terminology for the language in question.
+
+  - For each part-of-speech category:
+
+      - What is the name of this category?
+
+      - What abbreviations are typically used and/or preferred to refer to
+        this category?
+
+      - A prose description of the category.
+
+      - What are some example members of this category?
+
+      - Are the members declinable?
+
+        If the members of this category are in theory declinable, but for
+        practical purposes morphodict will not be doing any declining—maybe
+        nobody’s figured out the rules yet, or the rules are known but
+        nobody’s put them into an FST—it is useful to note that.
+
+      - Do all the members of this category share the same paradigm table
+        template, with the same FST tags but different FST lemma inputs?
+
+        If the answer to this question is ‘no,’ morphodict cannot use this
+        particular part-of-speech category as a paradigm, and you can skip
+        the other questions for this category.
```

Comment:
This is basically what Arok did for Plains Cree—he divided the categories into subcategories based on the details of how they inflect.

```suggestion
        If the answer to this question is ‘no,’ morphodict cannot use this
        particular part-of-speech category as a paradigm, and you can skip
        the other questions for this category. You could however create
        subcategories, each with their own inflectional paradigm.
```